app/src/main/java/com/eveningoutpost/dexdrip/utils/ActivityWithMenu.java

```diff
@@ -44,11 +48,14 @@ protected void onResume(){
     public void onNavigationDrawerItemSelected(int position) {
         if (position != menu_position) {
             startActivity(intent_list.get(position));
-            finish();
+            Set<String> categories = getIntent().getCategories();
+            getIntent().getAction();
+            //do not close activity if it is the Launcher or "Home".
+            if(categories == null || ! (categories.contains("android.intent.category.LAUNCHER") || getMenuName().equalsIgnoreCase(Home.menu_name))){
```

Comment:
I believe that writing it like this will be more clear. Only adding another () before not (no real change though):
if(categories == null || ! ((categories.contains("android.intent.category.LAUNCHER") || getMenuName().equalsIgnoreCase(Home.menu_name))))